### DISCUSSION TITLE
gh repo clone does not work

### DISCUSSION THREAD

**Discussion by maku1958** [community/community#27357] [New to GitHub]
Hello 
I'm frustrated now.
I have several times registered my device. Now I figured probably out that the device is wrong. I'm working via ssh on my homeserver, there should the gh repo clone be distributed.
But the registering is performed on my remote Computer.
Any Idea how to register my Server, which has no GUI?

Thanks in advance
maku1958

**Comment by airtower-luna** [comment]
> Any Idea how to register my Server, which has no GUI?

[`gh login` has a `--with-token` option](https://cli.github.com/manual/gh_auth_login) to let you provide a token instead of doing browser-based authorization. The minimum scopes are described in the linked docs, see [Creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for how to create the token.

**Comment by maku1958** [comment]
Thank you, that worked. --Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Le 27/07/2022, 21:11 Airtower ***@***.***> a écrit:

   
   
   Any Idea how to register my Server, which has no GUI? 
   
  gh login has a --with-token option to let you provide a token instead of doing browser-based authorization. The minimum scopes are described in the linked docs, see Creating a personal access token for how to create the token.

**Reply by airtower-luna** [reply]
I'm glad to hear that! Please mark the solution when you have a moment. :slightly_smiling_face:
